docs/integrations/datadog-rum/installation.md

```diff
@@ -0,0 +1,30 @@
+---
+id: installation
+title: Datadog RUM Installation
+sidebar_label: Installation
```

Comment:
Should this be linked in `sidebars.json`? It won't show up in the navigation otherwise.

Reply:
Would you do the honors?